docs/faq.md

```diff
@@ -100,12 +127,88 @@ Additionally, `ansible-navigator` will automatically volume mount the user's SSH
+$ cd  ./collections/ansible_collections/ansible/utils/
+$ ansible-navigator exec -- ansible-test sanity --python 3.8
+```
+
+### How do I use `ansible-playbook` parameters like `--forks 15`?
+
+All parameters not directly used by `ansible-navigator` will be passed to the `ansible-playbook` command. These can be provided inline after the `ansible-navigator` parameters or delimited by a `--`
+
+```bash
+$ ansible-navigator run site.yml --forks 15
+$ ansible-navigator run site.yml -- --forks 15
+```
+
+### How can I use a vault password with `ansible-navigator`?
+
+The following options provide a vault password to `ansible-navigator` when using the text-based user interface (TUI). **Please ensure these do not conflict with your enterprise security standards. Do not add password files to source control.**
+
+1. Store the vault password securely on the local file system
+
+```bash
+$ touch ~/.vault_password
+$ chmod 600 ~/.vault_password
+# The leading space here is necessary to keep the command out of the command history
+$  echo my_password >> ~/.vault_password
+# Link the password file into the current working directory
+$ ln ~/.vault_password .
+# Set the environment variable to the location of the file
+$ ANSIBLE_VAULT_PASSWORD_FILE=.vault_password
+$ ansible-navigator run site.yml
+```
 
-### Why does the playbook hang when `vars_prompt` or `pause/prompt` is used?
+2. Store the vault password in an environment variable
+
+```bash
+$ touch ~/.vault_password.sh
+$ chmod 700 ~/.vault_password.sh
+$ echo -e '#!/bin/sh\necho ${ANSIBLE_VAULT_PASSWORD}' >> ~/.vault_password.sh
+# Link the password file into the current working directory
+$ ln ~/.vault_password.sh .
+# The leading space here is necessary to keep the command out of the command history
+# by using an environment variable prefixed with ANSIBLE it will automatically get passed
+# into the execution environment
+$  export ANSIBLE_VAULT_SECRET=my_password
+# Set the environment variable to the location of the file
+$ ANSIBLE_VAULT_PASSWORD_FILE=.vault_password.sh
+$ ansible-navigator run site.yml
+```
+
+Additional information about `ansible-vault` can be found [here](https://docs.ansible.com/ansible/latest/user_guide/vault.html)
 
-By default `ansible-navigator` runs the playbook in the same manner that ansible controller and AWX would run the playbook. This was done to help playbook developers author playbooks that would be ready for production. If the use of `vars_prompt` or `pause\prompt` can not be avoided, disabling `playbook-artifact` creation and using `--mode stdout` causes `ansible-navigator` to run the playbook in a manner that is compatible with `ansible-playbook` and allows for user interaction.
+## Other
 
 ### Something didn't work, how can I troubleshoot it?
 
 `ansible-navigator` has reasonable logging messages, debug logging can be enabled with `--log-level debug`. If you think you might have found a bug, please log an issue and include the details from the log file.
```

Comment:
We should link "log an issue" to the new issue page

Reply:
will do